UPGRADING.rst

```diff
@@ -11,6 +11,17 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#4190 Create SNS topic for monitoring and security notifications
+================================================================
+
+A new environment variable ``AZUL_NOTIFICATION_EMAIL`` has been added. In
+personal deployments, set this variable to copy its value from ``AZUL_OWNER``.
```

Comment:
Confusingly written. It could mean that I need to copy the value manually or that I set it to `{AZUL_OWNER}`.